reconcile/utils/terrascript_client.py

```diff
@@ -1549,6 +1549,77 @@ def populate_tf_resource_s3(self, resource, namespace_info):
                 sqs_identifier, **notification_values)
             tf_resources.append(notification_tf_resource)
 
+        event_notifications = common_values.get('event_notifications')
+        sns_notifications = []
+        sqs_notifications = []
+
+        for event_notification in event_notifications:
+            destination_type = event_notification['destination_type']
+            detination_identifier = event_notification['destination']
+            event_type = event_notification['event_type']
+
+            if destination_type == 'sns':
+                notification_type = 'topic'
+                resource_arn_data = 'data.aws_sns_topic'
+            elif destination_type == 'sqs':
+                notification_type = 'queue'
+                resource_arn_data = 'data.aws_sqs_queue'
+
+            if detination_identifier is not None:
+                identifier_is_arn = True \
```

Comment:
Not sure if Im overlooking something here, but I think the `if else` is redundant. The following should produce the same

```
identifier_is_arn = destination_identifier.startswith('arn:')
```

Reply:
Good catch!